### PR TITLE
Disable Admin Feature

### PR DESCRIPTION
**Pull request by complacentsee**
Added ability to disable admin features. Addresses issue #51

### CODE REVIEW

**Comment by complacentsee** [comment]
I need to reproof this gitdiff.

**Comment by complacentsee** [comment]
I

**Comment by complacentsee** [comment]
Fixed error on admin mode labeled 'Night-Mode'.

File: templates/showforum.tpl [hunk 1]
```diff
@@ -12,6 +12,12 @@
   | <a href="#post"><b>Post New Thread</b></a>
   | <a href="#" id="view-all-images">View All Images</a>
   | <a href="#" id="night-mode">Night Mode: <span id="night-mode-status">Off</span></a>
+  <div style="display: inline"> | </div>
+<<<<<<< d37bde8b21c7f9263567902ecdd2f459138e51f6
+  <a href="#" id="admin-mode">Night Mode: <span id="admin-mode-status">Disabled</span></a>
+=======
+  <a href="#" id="admin-mode">Admin Mode: <span id="admin-mode-status">Disabled</span></a>
+>>>>>>> Disable Admin Feature
```
Comment (nyetwurk) [inline templates/showforum.tpl:20]: :)

Comment (complacentsee) [reply templates/showforum.tpl]: Some day I'll learn how to read diffs the first time.

File: templates/showforum.tpl [hunk 2]
```diff
@@ -12,6 +12,9 @@
   | <a href="#post"><b>Post New Thread</b></a>
   | <a href="#" id="view-all-images">View All Images</a>
   | <a href="#" id="night-mode">Night Mode: <span id="night-mode-status">Off</span></a>
+  <div style="display: inline"> | </div>
+  <a href="#" id="admin-mode">Night Mode: <span id="admin-mode-status">Disabled</span></a>
```
Comment (nyetwurk) [inline templates/showforum.tpl:16]: Admin Mode:

Comment (complacentsee) [reply templates/showforum.tpl]: Apparently I can't read today.

**Pull request merged**